client/my-sites/site-settings/custom-content-types/index.jsx

```diff
@@ -141,10 +107,18 @@ class CustomContentTypes extends Component {
 		const { translate } = this.props;
 		return (
 			<div>
-				<SectionHeader label={ translate( 'Custom Content Types' ) } />
+				<SectionHeader label={ translate( 'Custom content types' ) } />
 
 				<Card className="custom-content-types__card site-settings">
 					<FormFieldset>
+						<div className="custom-content-types__info-link-container site-settings__info-link-container">
+							<InfoPopover position={ 'left' }>
+								<ExternalLink href={ 'https://support.wordpress.com/custom-post-types/' } icon target="_blank">
+									{ translate( 'Learn more about Custom Content Types.' ) }
```

Comment:
A great suggestion, but I'm going with the dot version for consistency with the rest of the info icons. @rickybanister do you think we should have a dot here or not?